3-async-await/solution.js

```diff
@@ -20,17 +20,63 @@ the format is up to you
 
 8. as an extra challenge add Promise.race() and Promise.any(), and try to get the idea of what happens
 */
+const prices = require('./prices')
+const products = require('./products')
 
-function solution() {
-    // YOUR SOLUTION GOES HERE
 
-    // You generate your id value here
+const idGenerator = () => {
+    const id = Date.now().toString();
+    return parseInt(id.slice(id.length - 2));
+}
+
+const promiseAll = async (id)=> {
+    try{
+    const allPromise = Promise.all([
+        products(id),
+        prices(id),       
+    ])
+    
+        const [product,price] = await allPromise;
```

Comment:
You could also do this in one single action 😄 
```suggestion
    const [product,price]  = await Promise.all([
        products(id),
        prices(id),       
    ])
```